p2p/tests/sessions.rs

```diff
@@ -568,3 +568,50 @@ fn p2p_sessions_register_more_than_limit() {
     assert_eq!(sessions.get_num_outbound_sessions(), 3);
     assert_eq!(sessions.num_missing_outbound(), 0);
 }
+
+#[test]
+/// Check that peer addresses are considered "similar" if the first two octets of the IP are
+/// matching.
+fn p2p_peer_address_is_similar_to_inbound_session() {
+    // Create sessions struct
+    let mut sessions = Sessions::<String>::default();
+
+    let inbound_address_1 = SocketAddr::new(IpAddr::V4(Ipv4Addr::new(127, 0, 0, 1)), 8002);
```

Comment:
I prefer just parse and unwrap:
```suggestion
    let inbound_address_1 = "127.0.0.1:8002".parse().unwrap();
```

Reply:
Me too, but I simply copy-pasted from the rest of tests in the crate :rofl: